test-lib-openshift.sh

```diff
@@ -779,7 +779,6 @@ function ct_os_test_template_app_func() {
       for i_t in ${other_images} ; do
         echo "${i_t}"
         IFS='|' read -ra image_tag_a <<< "${i_t}"
-        docker pull "${image_tag_a[0]}"
```

Comment:
Just wondering if this change is expected as it does not seem to be related to the rest of the changes

Reply:
Good point. I wanted to remove docker pull, because of `docker pull` or especially pulling image has to be done at the beginning like here: https://github.com/sclorg/s2i-python-container/pull/449/files#diff-0b78990f6549655ad2c4497f3e0b771c74d42fc70837c7e1222728ae2cd566a4R20

But maybe we can add there a check if an image does not exist, then failed and say user to add ct_pull_image at the beginning of tests.

Reply:
See me proposal if it make sense to you.

Reply:
Makes sense, thanks!